public/upload.html

```diff
@@ -0,0 +1,96 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="s3.fine-uploader/fine-uploader-gallery.min.css" rel="stylesheet">
+    <script src="s3.fine-uploader/s3.fine-uploader.min.js"></script>
+    <script type="text/template" id="qq-template">
+        <div class="qq-uploader-selector qq-uploader qq-gallery" qq-drop-area-text="Drop files here">
```

Comment:
You sure you want all this HTML in the `head` tag instead of `body`?

Reply:
Should I put them in the body? Is that best practice?

Reply:
Yup: https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/The_head_metadata_in_HTML